debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9IndexableObjectHelper.java

```diff
@@ -124,22 +136,22 @@ public static long contiguousHeaderSize()
 	{
 		if (mixedReferenceMode) {
 			if (compressObjectReferences) {
-				return J9IndexableObjectContiguousCompressed.SIZEOF;
+				return J9IndexableObjectWithDataAddressContiguousCompressed.SIZEOF;
 			}
-			return J9IndexableObjectContiguousFull.SIZEOF;
+			return J9IndexableObjectWithDataAddressContiguousFull.SIZEOF;
 		}
-		return J9IndexableObjectContiguous.SIZEOF;
+		return J9IndexableObjectWithDataAddressContiguous.SIZEOF;
 	}
```

Comment:
I'm pretty bad at this, but I think it should go something like this:
- if contiguousIndexableHeaderSize in JavaVM exists (hence the core is from a new VM, with this change), we should read the size from that field
- if the field does not exist  (hence the core is from older VM), we should do whatever we used to do before.